src/main/main.ts

```diff
@@ -638,6 +641,12 @@ function handleRemoveServerModal(e: IpcMainEvent, name: string) {
     }
 }
 
+function updateSpellCheckerLocales() {
+    if (config.data?.spellCheckerLocales.length && app.isReady()) {
```

Comment:
if app is not ready, it will revert to system locale, right? wouldn't that feel weird?

Reply:
Yes, but then it runs this again on `initializeAfterAppReady`. I had to check that because there was a case where it would run before the app was ready and cause a crash.